src/testUtils/index.tsx

```diff
@@ -79,20 +160,309 @@ export const getSpecWrapper = (
     componentWrapper: ReactWrapper<{}, {}> | ShallowWrapper<{}, {}>,
     specName: string,
     attributeName: string = 'data-spec'
-): any => {
-    return componentWrapper.find(`[${attributeName}="${specName}"]`);
-};
+): any => componentWrapper.find(`[${attributeName}="${specName}"]`);
 
-const DEBUG = false;
+export const composeDuxState = (
+    query: Query<AppState | Partial<AppState>> = {},
+    duxState = baseState
+) => mutate(duxState, query);
 
-export const timeStart = (name: string) => {
```

Comment:
moved these to `/utils` since we were importing them into the app/not using them for tests